scripts/approvalYears.py

```diff
@@ -198,7 +201,7 @@ def apprDateRegression(prods):
     window = 10000
     dataxs = dict()
     datays = dict()
-    for prod in prods.keys():
+    for prod in list(prods.keys()):
```

Comment:
Same as above: "for prod in prods:"

Reply:
Python3 versus python2 issue. 2to3 automagically has made this change as .keys() no longer returns a list in python3